src/components/GetResilienceForm.js

```diff
@@ -0,0 +1,135 @@
+import React from 'react';
+import Container from '@material-ui/core/Container';
+import Grid from '@material-ui/core/Grid';
+import TextField from '@material-ui/core/TextField';
+import CssBaseline from '@material-ui/core/CssBaseline';
+import { makeStyles } from '@material-ui/core/styles';
+import { Typography, Button, Box } from '@material-ui/core';
+
+const useStyles = makeStyles((theme) => ({
+    space: {
+        height: theme.spacing(2)
+    },
+    padding: {
+        paddingTop: theme.spacing(3)
+    }
+}));
+
+const Form = () => {
+    const classes = useStyles();
+    return (
+        <>
```

Comment:
Haven't had time to look at material's components, do we need to wrap this all in a `<form>` ?

https://docs.netlify.com/forms/setup/

Also see netlify forms for form setup.

Do we also need to establish a route this will be rendered on? Just based of these changes it looks like we've made the component, but it won't be loaded anywhere

Reply:
I didnt added any route because on zendhub was another ticket for that. I know we need to wrap the compoment into a form, I'll add it now, thanks. I'll just check for the netlify forms

Reply:
Yup you should be able to just add `netlify` to the form element or `data-netlify="true"`, the docs cover it. But should be good to check with Daniel or Miles on what we're expecting

Reply:
Because we're using Gatsby, you'll need to add a hidden `input` like this guy:
`<input type="hidden" name="form-name" value="contact" />`
Per: https://www.netlify.com/blog/2017/07/20/how-to-integrate-netlifys-form-handling-in-a-react-app/#form-handling-with-static-site-generators

Reply:
Thank you

Reply:
@jessdwitch is the hidden input the same as a honeypot input that gatsby expects?

Ive barely dabbled with netlify forms and it seems like a perfect _free_ solution for what we needed with this contact form